mrp_tracking_label/README.md

```diff
@@ -0,0 +1,31 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+MRP Tracking Label
+==================
+
+Add a report with a label format.
```

Comment:
You should provide a basic layout for the report, although it can be customized.